### PR TITLE
Ejabberd UI documentation section

### PR DESCRIPTION
**Pull request by stephdl**
https://github.com/NethServer/dev/issues/5537

### CODE REVIEW

File: administrator-manual/en/ui/Ejabberd.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+:ref:`chat-section`.
+
+
+Administration web Interface
+    The Jaber server comes with a web administrative interface for members of the jabberadmins group.
+
+Federation (S2S)
+    XMPP allows for servers communicating seamlessly with each other, forming a global 'federated' IM network.
+
+File transfert maximum speed
```
Comment (gsanchietti) [inline administrator-manual/en/ui/Ejabberd.rst]: Multiple occurrences of :
transfert -> transfer

**Comment by stephdl** [comment]
corrected @gsanchietti

**Comment by gsanchietti** [comment]
There are still typos, check line 17 and 20.

File: administrator-manual/en/ui/Ejabberd.rst (hunk 1 above)
Comment (stephdl) [reply administrator-manual/en/ui/Ejabberd.rst]: bloody french, too close of english.....Normal we ruled the Britain so much years

Comment (gsanchietti) [reply administrator-manual/en/ui/Ejabberd.rst]: Aahahaha

Comment (gsanchietti) [reply administrator-manual/en/ui/Ejabberd.rst]: Aahahahahhahahahahaah

**Comment by stephdl** [comment]
should be good @gsanchietti sorry for the noise

**Pull request merged**